src/main/kotlin/interactor/GetBestCityForSavingMoneyInteractor.kt

```diff
@@ -0,0 +1,61 @@
+package interactor
+
+import model.CityEntity
+
+class GetBestCityForSavingMoneyInteractor(private val dataSource: CostOfLivingDataSource){
+
+    fun execute(isAppartmentInTheCityCentre: Boolean): CityEntity {
+
+        isAppartmentInCenter = isAppartmentInTheCityCentre
+
+        val cities = dataSource.getAllCitiesData()
+        var bestCity = cities[0]
+
+        for (city in cities) {
```

Comment:
should be replaced with kotlin functions e.g( **List.forEach{} , List.filter{}** )